include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -190,14 +191,23 @@ MjbotsControlLoop<log_type, input_type, robot_type>::MjbotsControlLoop(std::shar
     joints_moteus.push_back(
         std::dynamic_pointer_cast<kodlab::mjbots::JointMoteus>(j));
   }
+  // Open loop check
+  for(const auto & j : joints_moteus){
+      if(!options.open_loop && j->is_open_loop()){
+        LOG_FATAL("Open loop option not set, not receiving position and/or velocity");
+        kodlab::ActivateCtrlC();
+      }
+  }
   // Initialize mjbots_interface
   mjbots_interface_ = std::make_shared<kodlab::mjbots::MjbotsHardwareInterface>(
       std::move(joints_moteus), options.realtime_params,
       options.imu_mounting_deg, options.attitude_rate_hz,
       robot_->GetIMUDataSharedPtr(), options.imu_world_offset_deg,
       options.dry_run,
       options.print_torques,
-      options.send_pd_commands);
+      options.use_pd_commands,
+      options.open_loop
```

Comment:
Why does the `MjbotsHardwareInterface` take in the open_loop bool since it is not used in the class?